exercises/book-store/canonical-data.json

```diff
@@ -158,6 +158,16 @@
         "basket": [1,1,2,2,3,3,4,5,1,1,2,2,3,3,4,5]
       },
       "expected": 10240
+    },
+	{
+      "uuid": "5260ddde-2703-4915-b45a-e54dbbac4303",
+      "property": "total",
+      "description": "Check that groups of four are created properly even when there are more groups of three than groups of five.",
```

Comment:
This description is unique in this file in that it is the only one ending in a period.

This comment expresses no opinion on the presence or absence of periods.